mlflow/ml-package-versions.yml

```diff
@@ -161,11 +164,10 @@ catboost:
     install_dev: |
       # The cross-version-tests workflow runs this command with the environment variable `CACHE_DIR`
       if [ ! -d "$CACHE_DIR" ] || [ -z $(find $CACHE_DIR -type f -name "catboost-*.whl") ]; then
-        head_sha=$(git ls-remote https://github.com/catboost/catboost.git HEAD | cut -f1)
         pip wheel --no-deps --wheel-dir $CACHE_DIR \
-          git+https://github.com/catboost/catboost.git@$head_sha#subdirectory=catboost/python-package
+          git+https://github.com/catboost/catboost.git#subdirectory=catboost/python-package
```

Comment:
The latest version of `pip` automatically logs commit SHA when installing a package from a git repository: https://github.com/pypa/pip/pull/10149

Reply:
Are we running the latest version of `pip`?

Reply:
@dbczumar Yes! https://github.com/mlflow/mlflow/blob/c80598208c63ae74d87f2011bafb651c9a843fb7/.github/workflows/cross-version-tests.yml#L126